src/style.css

```diff
@@ -1,3 +1,68 @@
+ul,
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  display: flex;
+  align-items: center;
+}
+
+.menu__list {
+  display: flex;
+  align-items: center;
+}
+
+.menu__item {
+  margin-left: 20px;
+  list-style-type: none;
+}
+
+.menu__item:first-child {
+  margin-left: 0;
+}
+
+.menu__link {
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  display: block;
+}
+
+.menu__link:hover {
+  color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
I am confused, you told in the first comment to rename it to nav__link now there stands menu__link? I guess you mean nav__link right, please elaborate?
all tests passed